packages/keyda-demo/package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.19.2",
+    "lodash": "^4.17.15",
     "react": "^16.13.1",
```

Comment:
`axios` for submitting the form to our server, and `lodash` for checking inclusion.